bc_obps/registration/models.py

```diff
@@ -221,7 +222,15 @@ class Statuses(models.TextChoices):
     legal_name = models.CharField(max_length=1000, db_comment="The legal name of an operator")
     trade_name = models.CharField(max_length=1000, blank=True, db_comment="The trade name of an operator")
     cra_business_number = models.IntegerField(db_comment="The CRA business number of an operator")
-    bc_corporate_registry_number = models.IntegerField(db_comment="The BC corporate registry number of an operator")
+    bc_corporate_registry_number = models.CharField(
```

Comment:
not sure if this is necessary since I've already added validation to the schema

Reply:
No I think backend validation is a great idea as Django validates most of our fields already. Nice work, this is really cool and will be a good example if we need other types of custom backend validation going forward 🙇‍♂️